front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -119,16 +119,11 @@ const Map = ({
     bottom: 20,
   };
 
-  const [featureInfoClick, setFeatureInfoClick] = useState<FeatureInfoClick>({
-    displayPopup: false,
-  });
+  const [featureInfoClick, setFeatureInfoClick] = useState<FeatureInfoClick | undefined>();
```

Comment:
Undefined is set by default if you don't initialized your const
```suggestion
  const [featureInfoClick, setFeatureInfoClick] = useState<FeatureInfoClick>();
```

Reply:
Done